SwiftGit2.xcodeproj/project.pbxproj

```diff
@@ -739,12 +1179,227 @@
 		};
 		BEB31F951A0E56E200F525B9 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			target = BEB31F8E1A0E563900F525B9 /* libgit2 */;
+			target = BEB31F8E1A0E563900F525B9 /* libgit2-OSX */;
 			targetProxy = BEB31F941A0E56E200F525B9 /* PBXContainerItemProxy */;
 		};
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		621E66B21C72958800A0F352 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = BEB31F561A0D75EE00F525B9 /* Mac-Framework.xcconfig */;
```

Comment:
The iOS framework should have the iOS framework xcconfig as its base configuration. The iOS test framework should also have the iOS version of the configuration selected.

<img width="755" alt="configs" src="https://cloud.githubusercontent.com/assets/1302/13207165/3fa12db2-d8d9-11e5-8958-a47989f917d9.png">